### ISSUE TITLE
The notification for update is annoying.

### ISSUE THREAD

**Issue by zeakey** [nxshell/nxshell#168]
Thanks for developing such an excellent ssh tool.

As a user I have a suggestion to the software: the notification for updating appears every time I start the software, which annoys me a lot. I suggest having an option to disable the automatic update notification.

By the way, I think such a good tool should not be restricted in the Chinese community, but the issues here are full of Chinese content, which will limit the promotion to users worldwide.

**Comment by pistacio520** [comment]
About 2 suggestion, Is it possible to consider learning Chinese，

> Thanks for developing such an excellent ssh tool.
> 
> As a user I have two suggestions to the software:
> 
> 1. the notification for updating appears every time I start the software, which annoys me a lot. I suggest having an option to disable the automatic update notification.
> 2. The alert "are you sure to close the session" when closing a tab is also annoying. Even closing a disconnected session will triger such alert. The logical is quite weird because when I'm closing the entire software, there is no such alert, while the alert appears when I'm trying to close a single tab.
> 
> By the way, I think such a good tool should not be restricted in the Chinese community, but the issues here are full of Chinese content, which will limit the promotion to users worldwide.

**Comment by zeakey** [comment]
> About 2 suggestion, Is it possible to consider learning Chinese，
> 
> > Thanks for developing such an excellent ssh tool.
> > As a user I have two suggestions to the software:
> > 
> > 1. the notification for updating appears every time I start the software, which annoys me a lot. I suggest having an option to disable the automatic update notification.
> > 2. The alert "are you sure to close the session" when closing a tab is also annoying. Even closing a disconnected session will triger such alert. The logical is quite weird because when I'm closing the entire software, there is no such alert, while the alert appears when I'm trying to close a single tab.
> > 
> > By the way, I think such a good tool should not be restricted in the Chinese community, but the issues here are full of Chinese content, which will limit the promotion to users worldwide.

Yes actually I'm a native Chinese. I mean we may try to increase the impact of NxShell and attract more international users.
[mmdetection](https://github.com/open-mmlab/mmdetection) is a good example that.
